### DISCUSSION TITLE
BypassPropertyChecks flag not set to true in factory.

### DISCUSSION THREAD

**Discussion by kcabral817** [MarimerLLC/csla#2535] [Questions]
![image](https://user-images.githubusercontent.com/26425998/137675040-8bbb705c-67b0-4899-8f4e-100d0365523f.png)
In the above example code, I'm using a factory to map between csla object and sql server object.  I've set the suppress rule checking before automapper maps between the objects, but the isBypassingPropertyChecks flag is set to false.  Should the bypass flag be true and should it be bypassing for the object graph?

**Comment by rockfordlhotka** [comment]
These are two different things.

The `SuppressRuleChecking` turns off the rules engine entirely, while `BypassPropertyChecks` disables it for a specific business object.

I suppose, though, that it might make some sense for the flag to be consistent - as obviously this does seem inconsistent.
